### PR TITLE
Added function to calculate E(B-V) extinction effect

### PR DESCRIPTION
**Pull request by urstrulykkr**
Completed first objective of writing a function to calculate E(B-V). Waiting to get reviewed.

### CODE REVIEW

**Comment by urstrulykkr** [comment]
Why is Travis CI refraining ?

File: starkit/func_EBV.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import astropy.units as au
+
+def getEBV_effect(rtAs_n,dec_n):
+     '''
+     Generates E(B-V) i.e magnitude of Extinction effect with sky coordinates.
+     parameters
+     ----------
+     rightAscension: angular distance of a particular point measured eastward along equator
+     declination: angular distance of a point north or south of the celestial equator
+     frame:  system of metrics to calculate the magnitude of reddening (here)
```
Comment (followthesheep) [inline starkit/func_EBV.py]: what does 'frame' mean? This does not seem to be a parameter in the code

Comment (urstrulykkr) [reply starkit/func_EBV.py]: Yes, that makes sense. Will remove it.  Thanks @followthesheep :)

**Comment by urstrulykkr** [comment]
`@urstrulykkr I sent you a comment on your PR via github. It would be good now to think about how to apply the extinction model to a star’s spectrum`
As you have mentioned in Gitter, should I present it in a working model ? or in a description?

File: starkit/func_EBV.py (hunk 1 above)
Comment (urstrulykkr) [reply starkit/func_EBV.py]: Done :+1:

Comment (urstrulykkr) [reply starkit/func_EBV.py]: @followthesheep , May I know why did you mark the above as outdated?

**Comment by urstrulykkr** [comment]
@followthesheep I've tried to apply the extinction model to a star’s spectrum. Here applied to Gaussian and Black body spectrum. (Just experimented and learnt a lot :) )
Your reviews are welcomed.

**Comment by urstrulykkr** [comment]
Travis CI has some issues

**Comment by urstrulykkr** [comment]
**Travis CI has some issues here.**

**Pull request closed without merging**